CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 Current
+New: GITHUB-2897: Not exception but warning if some (not all) of the given test names are not found in suite files. (Bruce Wen)
```

Comment:
Can we please revert the formatting changes in this file ?

Reply:
Yes, to fix.